waku/v2/protocol/waku_store/waku_store.nim

```diff
@@ -459,24 +459,58 @@ proc query*(w: WakuStore, query: HistoryQuery, handler: QueryHandlerFunc) {.asyn
   waku_store_messages.set(response.value.response.messages.len.int64, labelValues = ["retrieved"])
   handler(response.value.response)
 
+proc queryFrom*(w: WakuStore, query: HistoryQuery, handler: QueryHandlerFunc, peer: PeerInfo): Future[bool] {.async.} =
```

Comment:
If queryFrom and resume are public functions, should they be exposed in node API at some point?

Reply:
Can you elaborate on exposing this proc in the node API? Which api?

Reply:
`wakunode` api https://github.com/status-im/nim-waku/blob/master/docs/api/v2/node.md

Reply:
Thanks for the link, I also agree that both `queryFrom` and `resume` must be exposed in node api. Will do it in subsequent PRs.

Reply:
Corresponding issue https://github.com/status-im/nim-waku/issues/557